content/en/docs/reference/glossary/security-context.md

```diff
@@ -17,9 +17,12 @@ including the runtime UID and GID.
 
 <!--more-->
 
-The `securityContext` field in a Pod or container is used to set the user,
-groups, capabilities, privilege settings, and security policies
-(SELinux/AppArmor/Seccomp) and more that container processes use.
+The `securityContext` field in a Pod or container defines the security controls
+and settings that apply to container processes.
+The PodSpec.securityContext setting applies to all containers in a Pod.
 
-If you set securityContext for a Pod, it applies to all the containers in
-that Pod.
+In a `securityContext`, you can define:
+- the user that processes run as
+- the group that processes run us
```

Comment:
s/us/as/

Reply:
@sftim , this PR is almost ready. Would you check this last change.
https://deploy-preview-18092--kubernetes-io-master-staging.netlify.com/docs/reference/glossary/?security=true
The short description is clear, but the `more` section somewhat duplicates the short description. When the full description is read, there is some content repeated. Also, not sure whether the list works well for a glossary entry. The list is not rendering (yet). Is there a typo on line 26?